testing/sdk_networks.py

```diff
@@ -60,3 +60,45 @@ def check_ip_addresses_on_overlay():
     for dns in endpoints["dns"]:
         assert "autoip.dcos.thisdcos.directory" in dns, "DNS {} is incorrect should have autoip.dcos.thisdcos." \
                                                         "directory".format(dns)
+
+
+def get_framework_srv_records(package_name):
+    cmd = "curl localhost:8123/v1/enumerate"
+    ok, out = shakedown.run_command_on_master(cmd)
+    assert ok, "Failed to get srv records. command was {}".format(cmd)
+    srvs = json.loads(out)
+    fmk_srvs = [f for f in srvs["frameworks"] if f["name"] == package_name]
```

Comment:
Maybe `framework_srvs`, otherwise it's super easy to read this as `framework_services`